tracer/src/Datadog.Trace/Propagators/ContextPropagationHeaderStyle.cs

```diff
@@ -0,0 +1,66 @@
+﻿// <copyright file="ContextPropagationHeaderStyle.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+namespace Datadog.Trace.Propagators;
+
+/// <summary>
+/// Values used to indicate propagation header styles.
+/// </summary>
+internal static class ContextPropagationHeaderStyle
+{
+    /// <summary>
+    /// The W3C tracecontext propagation header style.
+    /// Uses headers: traceparent, tracestate.
+    /// </summary>
+    public const string W3CTraceContext = "TRACECONTEXT";
+
+    /// <summary>
+    /// The origin Datadog propatation header style.
+    /// Uses headers: x-datadog-trace-id, x-datadog-parent-id, x-datadog-sampling-priority,
+    /// x-datadog-origin, x-datadog-tags.
+    /// </summary>
+    public const string Datadog = "DATADOG";
+
+    /// <summary>
+    /// The B3 propagation header style using multiple headers.
+    /// Uses headers: X-B3-TraceId, X-B3-SpanId, X-B3-Sampled.
+    /// </summary>
+    public const string B3MultipleHeaders = "B3MULTI";
+
+    /// <summary>
+    /// The B3 propagation header style using a single header.
+    /// Uses headers: b3.
+    /// </summary>
+    public const string B3SingleHeader = "B3 SINGLE HEADER";
```

Comment:
FYI, at some point in the future we'll want the user to specify the string `b3` to select the B3SingleHeader propagation method. Perhaps if we add telemetry to the values we observe, we can be more informed on when we can do the switch